### ISSUE TITLE
GHA invest-autotest job does not fail when a model errors

### ISSUE THREAD

**Issue by davemfish** [natcap/invest#1468]
Here's a recent log from this Action, which runs on `main`. Note the GHA step does not show a failure. https://github.com/natcap/invest/actions/runs/7170463471/job/19523281269   (an unrelated step also fails for an unrelated reason in this example)

```
********FAILURES********
MODELNAME                         EXIT CODE  DATASTACK                                                                           WORKSPACE
hra                               1          data/invest-sample-data\HabitatRiskAssess/hra_wcvi.invs.json                        C:\Users\RUNNER~1\AppData\Local\Temp\tmphbgykk8x\autorun_hra_0
routedem                          1          data/invest-sample-data\RouteDEM/routedem_gura.invs.json                            C:\Users\RUNNER~1\AppData\Local\Temp\tmphbgykk8x\autorun_routedem_0
```

**Comment by davemfish** [comment]
We didn't fix this the first time. I think the new problem is the `continue-on-error` flag.

**Comment by davemfish** [comment]
Fixed by #1476
